apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts

```diff
@@ -288,11 +282,21 @@ onDocumentReady(() => {
 });
 
 async function ajaxSubmit(this: HTMLFormElement, e: SubmitEvent) {
+  const formData = new FormData(this, e.submitter);
+
+  // Access specific values from the form data (or other fields)
+  const action = formData.get('__action');
+  const batchAction = formData.get('batch_action');
+
+  if (action === 'batch_action' && batchAction === 'ai_grade_assessment_selected') {
+    return;
+  }
```

Comment:
```suggestion
  if (action === 'batch_action' && batchAction === 'ai_grade_assessment_selected') {
    // We'll handle this with a normal form submission since it redirects to another page.
    return;
  }
```